Use native-image 22.2.0

julienrf opened the pull request:
I had to bump the version of native-image to be able to use it in sbt/sbtn-dist#4.

julienrf committed:
Use native-image 22.3.1

lightbend-cla-validator commented:
Hi @julienrf,

Thank you for your contribution! We really value the time you've taken to put this together.

We see that you have signed the Lightbend Contributors License Agreement before, however, the CLA has changed since you last signed it.
Please review the new CLA and sign it before we proceed with reviewing this pull request:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

julienrf commented:
Do you know if the CI failure could be related to my change?

julienrf committed:
Downgrade to 22.2.0

For some reason the jvm for 22.3.1 could not be found by Coursier, although it is present in the index.

julienrf commented:
I signed the CLA
